k-distribution/include/kframework/builtin/domains.md

```diff
@@ -1678,7 +1678,7 @@ module STRING-KORE [kore, symbolic]
   imports private K-EQUAL
   imports STRING-COMMON
 
-  rule S1:String ==K S2:String => S1 ==String S2 [simplification]
+  rule S1:String ==K S2:String => S1 ==String S2 [simplification, comm]
```

Comment:
```suggestion
  rule S1:String ==K S2:String => S1 ==String S2 [simplification]
```

Reply:
This rule already is commutative, and doesn't need a duplicate of it generated for the other direction.